scripts/hera_mini_snap_data_catcher.py

```diff
@@ -278,9 +294,18 @@ def get_blt_index(antnum, time, ant_array, time_array):
                 uvd.extra_keywords["snap_to_ant_mapping"] = json.dumps(
                     snap_to_ant_mapping
                 )
-                fname = f"zen.{Time.now().jd:.6f}.snap_autos.uvh5"
+
+                date = Time.now().jd
+
+                folder_name = f"{int(np.floor(date)):d}"
+
+                # make the fold for the JD if it does not exist
+                folder = Path(args.outdir) / folder_name
```

Comment:
I assume this works, I would normally use `os.path.join` myself.

Reply:
Yeah I like the object based interface for the pathlib.Path object. It does resolve to `os.path.join` eventually I bet.